x-pack/plugins/data_visualizer/public/application/common/components/document_count_content/document_count_chart/document_count_chart.tsx

```diff
@@ -48,9 +46,12 @@ export const DocumentCountChart: FC<Props> = ({
   interval,
 }) => {
   const {
-    services: { data },
+    services: { data, uiSettings, fieldFormats },
   } = useDataVisualizerKibana();
 
+  const xAxisFormatter = fieldFormats.deserialize({ id: 'date' });
```

Comment:
As discussed, is it possible to have the time formatting the same in the tooltip and the tick labels? I wonder if they should use the default format for date fields, or even the format configured for the timestamp field in the data view?

![image](https://user-images.githubusercontent.com/7405507/140337126-db15e930-af6c-4ff2-b337-909369c4ac58.png)

Reply:
The tooltip uses the format configured under `dateFormat` in Advanced settings, so the current behavior is probably ok. Not sure if / how the top time line of the multiline axis would use that setting?